SU-SWS/decanter
# More card variants with different media options - image icon, video, embed

# READY FOR REVIEW

# Summary
- Create variants of cards that work with these new media types - videos, embeds, icons (image).
- Media portion of card now uses the Decanter Media component.
- New variants should work with existing card variants (horizontal, minimal, link).
- Card content and button should be left aligned except for the icon variant.
- Use `@small-paragraph` for the card body content.
- When the button is the last element in the card, add extra margin at the bottom for optical balance.

# Needed By (Date)
- Whenever

# Urgency
- Not urgent

# Steps to Test

1. Pull this branch and compile styleguide.
2. Look at the new variants - `su-card--icon, su-card--video, su-card--embed` and see if they behave as intended.
3. Check that the new variants works with the `--minimal, --horizontal, --link` card variant classes.
Note: The icon variants shows the icon at xs breakpoints since they have small file size.
4. Check that the Hero component has the same new style tweaks as Cards but otherwise behave the same (it has a nested Card component).

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- #435 
- #431 
- #338 
- #455
